irida_staramr_results/cli.py

```diff
@@ -0,0 +1,69 @@
+import sys
+import argparse
+import logging
+from version import __version__
+from irida_api import IridaAPI
+from urllib.error import HTTPError
+
+
+def init_argparser():
+    argument_parser = argparse.ArgumentParser(
+        prog="irida-staramr-results",
+        description="Exports StarAMR results available through IRIDA into a single excel report."
+    )
+    argument_parser.add_argument("-v", "--version", action="version", version=f"{argument_parser.prog} {__version__}",
+                                 help="The current version of irida-staramr-results.")
+    argument_parser.add_argument("-p", "--project", action="store",
+                                 help="Project(s) to scan for StarAMR results.")
+    argument_parser.add_argument("-o", "--output", action="store",
+                                 help="The name of the output excel file.")
+
+    # Arguments that will override fields in the config file.
+    argument_parser.add_argument("-u", "--username", action="store",
+                                 help="Override the 'username' in config file. This is your IRIDA account username.")
```

Comment:
We are not overriding a username/password in the config file. The config file should have client id and client secret but not username/password.

Reply:
Done, removes misleading sentence. Now it only says "This is your IRIDA account username/password."

Reply:
Oops, I forgot to remove a comment. I did another push that removed it.